src/types.ts

```diff
@@ -258,3 +258,5 @@ export type DeepPartial<T> = {
       ? ReadonlyArray<DeepPartial<U>>
       : DeepPartial<T[P]>
 };
+
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
```

Comment:
Omit types, e.g.

```ts
interface A {
    a: number;
    b: number;
}

type B = Omit<A, 'b'>

// type B = {
//    a: number;
//}
```

Reply:
I believe TS 3.5 added a built in Omit type, so once you merge master into v4 I think this will complain about conflicts.

We still want to support consumers on earlier versions so I think we want to keep this around but under a different name for the moment

Reply:
> I believe TS 3.5 added a built in Omit type

🎉 

> We still want to support consumers on earlier versions so I think we want to keep this around but under a different name for the moment

👍

Reply:
Omit is pretty trivial to implement on your own. I don't feel this is necessary to export.